### ISSUE TITLE
Xray issue that I don't see listed here

### ISSUE THREAD

**Issue by BogglyBear** [fr1kin/ForgeHax#447]
I'm on 1.16.5, and was using the xray feature fine last night, but I load it up today and am greeted with bugged xray where I see almost all the blocks, so its pretty useless at that point for obvious reasons. So I try and clear the list then readd the ores, and it didn't work. I then tried adding stone to the list and removing and it didn't do nothing either.

**Comment by fr1kin** [comment]
Try installing OptiFine or just using markers instead

**Comment by BogglyBear** [comment]
I have optifine on, i'm just confused as to why it randomly did that

**Comment by BogglyBear** [comment]
funny thing is xray looked fine when i took out optifine wtf lol

**Comment by fr1kin** [comment]
i dont think it will work if you enable shaders on optifine

**Comment by BogglyBear** [comment]
Yeah i figured that one out and it was the issue haha, turned off the shaders and it works nicely
